cfgov/agreements/jinja2/agreements/index.html

```diff
@@ -51,6 +51,11 @@ <h2 id="issuer-select">
                     for agreement submission instructions.
                 </p>
                 <ul class="m-list m-list__links u-mt15 cc-links">
+                    <li class="m-list m-list__links">
```

Comment:
Hmmmmmmmm 

I know you're just copying what's already here, but these link structures are wrong.

I've opened https://github.cfpb.gov/CFGOV/platform/issues/4093

Reply:
Thank you!